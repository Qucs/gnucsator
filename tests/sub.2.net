# Qucs 0.0.19  /tmp/neur.sch

.Def:TESTSUB _net0 _net1 RES="1" PARAM2="2"
resistor:a _net0 _net1 r="RES"
.Def:End

.DC:DC1 Temp="26.85" reltol="0.001" abstol="1 pA" vntol="1 uV" saveOPs="no" MaxIter="150" saveAll="no" convHelper="none" Solver="CroutLU"
Sub:T1 _net1 0 Type="TESTSUB" RES="2"
Idc:V1 _net1 0 I="5"
