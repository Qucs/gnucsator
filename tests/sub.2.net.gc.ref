<Qucs Dataset 0.0.19>
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
<indep _net1.V 1>
10.
</indep>

<indep time 0>
</indep>
