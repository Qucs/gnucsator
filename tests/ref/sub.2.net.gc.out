<Qucs Dataset>
default plugins: ignored
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
store: already installed, replacing
stashing as store:0
alarm: already installed, replacing
stashing as alarm:0
iplot: already installed, replacing
stashing as iplot:0
plot: already installed, replacing
stashing as plot:0
iprint: already installed, replacing
stashing as iprint:0
print: already installed, replacing
stashing as print:0
probe: already installed, replacing
stashing as probe:0
L: already installed, replacing
stashing as L:0
R: already installed, replacing
stashing as R:1
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
parameter TESTSUB not specified, using default
parameter TESTSUB value is "NOT_INPUT"
Gnucap   System status
iterations: op=2, dc=2, tran=0, fourier=0, total=4
transient timesteps: accepted=0, rejected=0, total=0
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%

<indep _net1.V 1>
 10.
</indep>
